pfcpiface/main.go

```diff
@@ -6,14 +6,14 @@ package main
 import (
 	"encoding/json"
 	"flag"
+	fqdn "github.com/Showmax/go-fqdn"
 	"io/ioutil"
 	"log"
 	"net"
 	"net/http"
+	_ "net/http/pprof"
```

Comment:
http://128.105.146.13:33000/debug/pprof/
Open Nodeport 33000. Nodeport service maps 33000 to 8000 internally...
Restserver started for prometheus KPIs is used for profile purpose.

Reply:
Not sure if we should have this in the code. If we choose to have it then can we conditionally enable it? I thought here we have such option

https://github.com/pkg/profile#usage

Reply:
Whatever you are referring needs to be started/stopped to get the trace. Which means we can do it based on some rest APIs. It generates .pprof file. How do we see those pprof file ?

Reply:
Like this you should be able view the results in browser at host:port - 

profile
```
go tool pprof --http <host:port> <binary> <filename.pprof>
```

trace
```
go tool trace --http <host:port> <binary> <trace.out>

Reply:
@krsna1729 but it needs trigger to enable/disable profile.

Reply:
the pkg/profile version? No you are programmatically calling the API to start and stop. I tried the example in the link above